src/lib/components/Container.svelte

```diff
@@ -1,3 +1,5 @@
-<section class="prose p-4 lg:prose-xl lg:p-10" role="main">
-	<slot />
+<section class="prose p-4 lg:prose-xl lg:p-10">
+	<main id="main-content">
```

Comment:
IMHO main should be around sections and articles
- https://developer.mozilla.org/en-US/docs/Web/HTML/Element/main?retiredLocale=de
- https://www.w3schools.com/tags/tag_main.asp

Reply:
See 7a2f991.